src/react-native/18-maps/maps.md

```diff
@@ -28,11 +30,15 @@ TODO
 npm install react-native-maps@1.15.1
 ```
 
-✏️ Create **android/local.properties** and update it to be:
+✏️ Create **android/local.defaults.properties** and update it to be:
+
+@sourceref ../../../exercises/react-native/18-maps/01-problem/android/local.defaults.properties
 
-@sourceref ../../../exercises/react-native/18-maps/01-problem/android/.local.properties.example
+✏️ Create **android/secrets.properties** and update it to be:
 
-✏️ Update **GOOGLE_MAPS_API_KEY** with your key.
+@sourceref ../../../exercises/react-native/18-maps/01-problem/android/.secrets.properties.example
+
+✏️ Update **GOOGLE_MAPS_API_KEY** in `secrets.properties` with your key.
 
 ✏️ Update **android/build.gradle** to be:
```

Comment:
![Screenshot 2024-05-16 at 3 28 38 PM](https://github.com/bitovi/academy/assets/9858612/1270ed4a-0ef0-41f5-9bfe-6e408d126937)

we should note/hint this in the Exercise instructions

Reply:
a small nitpick, should this be:

![Screenshot 2024-05-16 at 3 29 49 PM](https://github.com/bitovi/academy/assets/9858612/2530dbd9-d471-4969-b322-6fd9479fd53b)

```
<MapView
   ...
/>
```
?

Reply:
can we provide these so they don't have to create them; or it will cause errors when they run the app?

![Screenshot 2024-05-16 at 3 31 35 PM](https://github.com/bitovi/academy/assets/9858612/9d7aac1e-1355-4fba-8947-4d113810a617)

![Screenshot 2024-05-16 at 3 31 41 PM](https://github.com/bitovi/academy/assets/9858612/97c493bf-0372-4776-aa4c-175e926e6a3f)